<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitForm</class>
 <widget class="QDialog" name="UnitForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unit Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="basicInfoGroupBox">
     <property name="title">
      <string>Basic Information</string>
     </property>
     <layout class="QFormLayout" name="basicFormLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="buildingLabel">
        <property name="text">
         <string>Building: *</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="buildingComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="unitNumberLabel">
        <property name="text">
         <string>Unit Number: *</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="unitNumberEdit">
        <property name="placeholderText">
         <string>e.g., 101, A-5, etc.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="floorLabel">
        <property name="text">
         <string>Floor:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="floorSpinBox">
        <property name="minimum">
         <number>-5</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="bedroomsLabel">
        <property name="text">
         <string>Bedrooms:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="bedroomsSpinBox">
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="bathroomsLabel">
        <property name="text">
         <string>Bathrooms:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="bathroomsSpinBox">
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="squareFeetLabel">
        <property name="text">
         <string>Square Feet:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="squareFeetSpinBox">
        <property name="maximum">
         <double>99999.989999999997672</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="financialGroupBox">
     <property name="title">
      <string>Financial Information</string>
     </property>
     <layout class="QFormLayout" name="financialFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="rentLabel">
        <property name="text">
         <string>Monthly Rent:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="rentSpinBox">
        <property name="prefix">
         <string>$ </string>
        </property>
        <property name="maximum">
         <double>999999.989999999990687</double>
        </property>
        <property name="singleStep">
         <double>50.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="tenantGroupBox">
     <property name="title">
      <string>Tenant Information</string>
     </property>
     <layout class="QFormLayout" name="tenantFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="statusComboBox">
        <item>
         <property name="text">
          <string>Vacant</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Occupied</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Maintenance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Reserved</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="tenantNameLabel">
        <property name="text">
         <string>Tenant Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="tenantNameEdit">
        <property name="placeholderText">
         <string>Tenant name (if occupied)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="leaseStartLabel">
        <property name="text">
         <string>Lease Start:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="leaseStartEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="leaseEndLabel">
        <property name="text">
         <string>Lease End:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateEdit" name="leaseEndEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="notesGroupBox">
     <property name="title">
      <string>Additional Notes</string>
     </property>
     <layout class="QVBoxLayout" name="notesLayout">
      <item>
       <widget class="QPlainTextEdit" name="notesEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Additional notes or comments about this unit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="requiredLabel">
     <property name="text">
      <string>* Required fields</string>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UnitForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>574</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UnitForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>574</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
